api-2.0/src/test/java/org/openmrs/module/fhir/api/ConditionServiceTest.java

```diff
@@ -64,4 +74,52 @@ public void getConditionByObsId_shouldReturnMatchingFHIRCondition() {
 		assertEquals(fhirCoding.getCode(), problemAddedObs.getConcept().getUuid());
 		assertEquals(fhirCoding.getDisplay(), problemAddedObs.getConcept().getName().getName());
 	}
+
+	@Test
+	public void createFHIRCondition() {
+		Context.getAdministrationService().saveGlobalProperty(new GlobalProperty(FHIRConstants
+				.CONCEPTS_CONVERTABLE_TO_CONDITIONS_STORED_AS_OBS, GLOBAL_PROPS_CONDITION_MAPPING_CONCEPT_ID));
+		Condition condition = new Condition();
+		String uuid = "75f5b373-5065-11de-80cb-001e378eb67f";
```

Comment:
Did you take a look at our naming conventions for constants? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions

Reply:
Did you see the above?

Reply:
@corneliouzbett did you see the above question?